### PR TITLE
Allow disabling access to the DB and the Teams API.

### PR DESCRIPTION
**Pull request by alexeyklyukin**
Command-line options --nodatabaseaccess and --noteamsapi disable all
teams api interaction and access to the Postgres database. This is
useful for debugging purposes when the operator runs out of cluster
(with --outofcluster flag).

The same effect can be achieved by setting enable_db_access and/or
enable_teams_api to false.

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -87,6 +91,12 @@ func main() {
 	if configMapData["namespace"] == "" { // Namespace in ConfigMap has priority over env var
 		configMapData["namespace"] = podNamespace
 	}
+	if noDBAccess {
+		configMapData["enable_db_access"] = "false"
```
Comment (Jan-M) [inline cmd/main.go]: db => database ;)

File: cmd/main.go [hunk 2]
```diff
@@ -19,12 +19,16 @@ var (
 	podNamespace   string
 	configMapName  spec.NamespacedName
 	OutOfCluster   bool
+	noTeamsAPI bool
+	noDBAccess bool
```
Comment (Jan-M) [inline cmd/main.go]: DB => database, but this is personal preference

**Comment by alexeyklyukin** [comment]
👍

**Comment by ikitiki** [comment]
👍

**Pull request merged**